api/v1alpha1/jobset_types.go

```diff
@@ -76,7 +92,7 @@ type ReplicatedJob struct {
 	// Replicas is the number of jobs that will be created from this ReplicatedJob's template.
 	// Jobs names will be in the format: <jobSet.name>-<spec.replicatedJob.name>-<job-index>
 	// +kubebuilder:default=1
-	Replicas *int `json:"replicas,omitempty"`
```

Comment:
I changed fields with kubebuilder default marker to be concrete data types instead of pointers, since the defaulting didn't seem to be working with pointers.